core/src/com/unciv/ui/tutorials/TutorialMiner.kt

```diff
@@ -2,33 +2,33 @@ package com.unciv.ui.tutorials
 
 import com.badlogic.gdx.Gdx
 import com.badlogic.gdx.utils.Array
-import com.unciv.UncivGame
+import com.unciv.JsonParser
 import com.unciv.models.Tutorial
 
-class TutorialMiner {
+class TutorialMiner(private val jsonParser: JsonParser) {
 
     companion object {
         private const val TUTORIALS_PATH = "jsons/Tutorials/Tutorials_%s.json"
     }
 
-    fun getAllTutorials(language: String): Map<Tutorial, List<String>> {
+    fun getCivilopediaTutorials(language: String): Map<Tutorial, List<String>> =
+            getAllTutorials(language).filter { it.key.isCivilopedia }
+
+    fun getTutorial(tutorial: Tutorial, language: String): List<String> =
+            getAllTutorials(language)[tutorial] ?: emptyList()
```

Comment:
Please do not use the Elvis operator (for readability for non-Kotlin-native contributors)

Reply:
Ok)